lib/sinks/cmdInjectionSemanticDangerousPaths.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const cp = require('child_process');
+
+const pre = (str) => `<pre>${str}</pre>`;
+
+/**
+ * @param {string} input user input string
+ * @param {Object} opts
+ * @param {boolean=} opts.safe are we calling the sink safely?
+ * @param {boolean=} opts.noop are we calling the sink as a noop?
+ */
+module.exports['child_process.exec'] = async function exec(
+  input,
+  { safe = false, noop = false } = {}
+) {
+  if (safe) return 'SAFE';
+  if (noop) return 'NOOP';
+
+  return new Promise((resolve) => {
+    cp.exec("/bin/sh -c 'cat /tmp/foo.txt /etc/passwd'", (err, data) => {
```

Comment:
this doesn't take any data from request which is ok but what do we do in darpa-testing-config to test these?

Reply:
That's a good point. Would it be better to just take in data from the request so that we could have darpa-testing-config sending different commands that'll trigger this rule?

Reply:
it's fine just curious